pytest_invenio/fixtures.py

```diff
@@ -590,8 +591,8 @@ def _get_screenshots_dir():
     return directory
 
 
-@pytest.yield_fixture(scope='function')
-def location(db):
```

Comment:
I'm not sure if this will have consequences. It changes scope from function to module scope. The change is to allow tests that just need a location defined, to run faster because it's not destroyed on every test.